docs/content/practices/state_clearing.md

```diff
@@ -118,37 +119,41 @@ It's just a shell client and should be installed to the device.
     Despite the fact that it does the job, this solution looks overcomplicated:
 
     1. We need to install +2 different apk to each emulator
-    2. We delegate this job to the device instead of host machine. 
+    2. We delegate this job to the device instead of the host machine. 
     <br/>Devices are less reliable than host pc
 
 ##### Other solutions
 
 It's also possible to clear package data by
 using [3rd party test runners](https://android-ui-testing.github.io/Cookbook/practices/test_runners_review/), like
 Marathon, Avito-Runner or Flank. Marathon and Avito-Runner clear package data without an orchestrator. They delegate
-this logic to a host machine
+this logic to a host machine.
 
 ##### Conclusion<br/>
 
 These are pros/cons for an `orchestrator` and 3rd party test runners solution:
 
-➕ Does the job for us in 100% <br/>
+➕ Does 100% of the job for us<br/>
+➕ Avoid test failing in cascade due to the application process being killed<br/>
 <br/>
-➖ Slow execution _(can take 10+ seconds and depends on apk size)_ <br/>
```

Comment:
10+ was for ALL the tests together due to adb pm clear right? This I did not understand very well